deployments/charts/kuma-cp/templates/pre-install-patch-namespace-job.yaml

```diff
@@ -0,0 +1,85 @@
+{{- if .Values.patchSystemNamespace }}
+{{- $serviceAccountName := printf "%s-patch-ns-job" (include "kuma-cp.name" .) }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ $serviceAccountName }}
+  annotations:
+    "helm.sh/hook": "pre-install"
+    "helm.sh/hook-delete-policy": "hook-succeeded,hook-failed"
```

Comment:
Hmm does this guarantee a proper cleanup when charts are deleted?

Reply:
Yes, it should be deleted either when the `pre-install` hook (i.e. running that job) succeeds or fails. Are you seeing differently?